tests/test_her.py

```diff
@@ -314,64 +309,31 @@ def test_full_replay_buffer():
         replay_buffer_kwargs=dict(
             n_sampled_goal=2,
             goal_selection_strategy="future",
-            online_sampling=True,
-            max_episode_length=n_bits,
         ),
         gradient_steps=1,
         train_freq=4,
         policy_kwargs=dict(net_arch=[64]),
-        learning_starts=1,
-        buffer_size=20,
+        learning_starts=n_bits * n_envs,
```

Comment:
maybe we should add a warning on the her doc page? (if it's not there already)

Reply:
I've just added a error message instead, do you think it's enough?